betterdiscordctl

```diff
@@ -4,34 +4,32 @@ set -ueo pipefail
 shopt -s dotglob extglob nullglob
 
 # Constants
-VERSION=1.7.1
+VERSION=1.8.0
 SOURCE=$(readlink -f "${BASH_SOURCE[0]}")
 DISABLE_UPGRADE=
 
 # Options
 cmd=status
 verbosity=0
-scan=(/opt /usr/share /usr/lib64)
 flavors=('' canary ptb)
-discord=
 modules=
+bd_asar_file=
 bd_repo='https://github.com/rauenzi/BetterDiscordApp'
-bd_repo_branch=injector
-bd=
-copy_bd=
-snap=
+bd_repo_release=latest
 flatpak=
-nix=
+flatpak_bin=flatpak
+snap=
+snap_bin=snap
 upgrade_url='https://github.com/bb010g/betterdiscordctl/raw/master/betterdiscordctl'
 
 # Variables
 flavor=
 core=
 xdg_config=${XDG_CONFIG_HOME:-$HOME/.config}
 data=${XDG_DATA_HOME:-$HOME/.local/share}/betterdiscordctl
-snap_bin=snap
-flatpak_bin=flatpak
-nix_bin='nix-store'
+discord_config=
+bd_config=$xdg_config/BetterDiscord
+bd_asar=$bd_config/data/betterdiscord.asar
```

Comment:
This can't actually be defined here because if `--bd-asar-file` (which I want to rename to `--bd-asar`) is defined to another location, and flatpak or snap are not used, the asar file will not there.